pytest_tests/test_env.py

```diff
@@ -0,0 +1,140 @@
+import json
+import os
+import re
+import uuid
+from time import sleep
+
+import boto3
+import pexpect
+import pytest
+from botocore.config import Config
+
+from neofs_testlib.env.env import NeoFSEnv, NodeWallet
+from neofs_testlib.utils.wallet import get_last_public_key_from_wallet, init_wallet
+
+
+def _run_with_passwd(cmd: str, password: str) -> str:
+    child = pexpect.spawn(cmd)
+    child.delaybeforesend = 1
+    child.expect(".*")
+    child.sendline(f"{password}\r")
+    child.wait()
+    cmd = child.read()
+    return cmd.decode()
+
+
+@pytest.fixture
+def neofs_env(request):
+    if request.config.getoption("--load-env"):
+        neofs_env = NeoFSEnv.load(request.config.getoption("--load-env"))
+    else:
+        neofs_env = NeoFSEnv.simple()
+
+    yield neofs_env
+
+    if request.config.getoption("--persist-env"):
+        neofs_env.persist()
+    else:
+        if not request.config.getoption("--load-env"):
+            neofs_env.kill()
+
+
+@pytest.fixture
+def wallet() -> NodeWallet:
+    wallet_name = f"{str(uuid.uuid4())}.json"
+    wallet_path = os.path.join(os.getcwd(), wallet_name)
+    wallet_password = "password"
+    wallet_address = init_wallet(wallet_path, wallet_password)
+    return NodeWallet(path=wallet_path, address=wallet_address, password=wallet_password)
+
+
+@pytest.fixture
+def s3_creds(neofs_env: NeoFSEnv, zero_fee, wallet: NodeWallet) -> tuple:
```

Comment:
S3 as a part of base env?

Reply:
In this particular case - yes. But actually any kind of env is configured via this kind of classmethods or fixtures:

```
  @classmethod
    def simple(cls) -> "NeoFSEnv":
        neofs_env = NeoFSEnv()
        neofs_env.deploy_inner_ring_node()
        neofs_env.deploy_storage_node()
        neofs_env.deploy_s3_gw()
        return neofs_env
```

So we can have whatever we want. But for the purpose of this PR - to have a working example - I use s3 gate here.